src/main/java/com/revlearn/team1/model/CourseModule.java

```diff
@@ -0,0 +1,40 @@
+package com.revlearn.team1.model;
+
+import jakarta.persistence.*;
+import lombok.Data;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Data
+public class CourseModule {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String title;
+
+    @Column(columnDefinition = "TEXT")
+    private String description;
+
+    @OneToMany(mappedBy = "courseModule", cascade = CascadeType.ALL)
+    private List<ModulePage> modulePages = new ArrayList<>();
+
+    @OneToMany(mappedBy = "courseModule", cascade = CascadeType.ALL)
+    private List<Exam> exams = new ArrayList<>();
+
+    @ManyToOne
+    @JoinColumn(name = "course_id", nullable = false)
```

Comment:
IDK if you want to set nullable to True and add a TODO to change it once things are more complete? no idea if it will make testing/debugging easier